addons/mail/static/src/models/channel.js

```diff
@@ -60,6 +60,20 @@ registerModel({
             }
             return clear();
         },
+        /**
+         * @private
+         * @returns {Partner|FieldCommand}
+         */
+        _computeCorrespondentOfDmChat() {
+            if (
+                this.channel_type === 'chat' &&
+                this.correspondent &&
+                this.thread.public === 'private'
```

Comment:
this condition is useless, and will be removed soon in another PR

Reply:
the check on `public === 'private'`? why? are chat channels always private?